### PR TITLE
Adds advanced mop and research

### PR DESCRIPTION
**Pull request by Naive817**
## About the PR

Janitors' new weapon!
Stats: 
+100% liquid capacity (50u->100u)
+150% liquid per mop (10->25u)
+25% mop speed (10->12.5)
+Can mop up to 3 puddles at a time
-- No weilding 

fab cost: plastic 100 steel 100 glass 100

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
屏幕截图 2023-03-28 103729
屏幕截图 2023-03-28 104705

https://user-images.githubusercontent.com/31364560/228116958-f2ae0e38-8616-4f74-8ae6-8d9b20d72a9e.mp4

**Changelog**

:cl: furontier
- add: NT brand advanced mops can now be researched.

### CODE REVIEW

File: Resources/Prototypes/Recipes/Lathes/janitorial.yml [hunk 1]
```diff
@@ -48,3 +48,12 @@
...
     Glass: 1000
+
+- type: latheRecipe
+  id: AdvMopItem
+  result: AdvMopItem
+  completetime: 2
+  materials:
+    Plastic: 100
+    Steel: 100
+    Glass: 100
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Recipes/Lathes/janitorial.yml:59]:
```suggestion
    Plastic: 400
    Steel: 100
    Glass: 100
```
rookie numbers (slightly too low imo)

**Review by EmoGarbage404 — APPROVED** [review]
lgtm

**Comment by Emisse** [comment]
could stand to make it hold more than 100u, for a research item you can make it a bit better

**Comment by Emisse** [comment]
eh actually i think this is good

**Pull request merged**